lib/mix/lib/mix/tasks/archive.install.ex

```diff
@@ -38,7 +38,7 @@ defmodule Mix.Tasks.Archive.Install do
 
       case Path.extname(path) do
         ".ez" -> install_archive(src, opts)
-        _     -> Mix.raise "mix archive.install doesn't know how to install #{path}"
+        _     -> Mix.raise "`mix archive.install` doesn't know how to install '#{path}'"
```

Comment:
Let's not use single quotes. The best here is to use `#{inspect path}` which also hands escape.